frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -30,6 +19,28 @@ export const getCephPVs = (pvsData: K8sResourceKind[] = []): K8sResourceKind[] =
     );
   });
 
+const getPVStorageClass = (pv: K8sResourceKind) => _.get(pv, 'spec.storageClassName');
+const getStorageClassName = (pvc: K8sResourceKind) =>
+  _.get(pvc, ['metadata', 'annotations', 'volume.beta.kubernetes.io/storage-class']) ||
+  _.get(pvc, 'spec.storageClassName');
+const isBound = (pvc: K8sResourceKind) => _.get(pvc, 'status.phase') === 'Bound';
+
+export const getCephPVCs = (
+  cephSCNames: string[] = [],
+  pvcsData: K8sResourceKind[] = [],
+  pvsData: K8sResourceKind[] = [],
```

Comment:
nit: I prefer `pvcData` and `pvData` since data is already plural and `pvs` could be confused with `pvc`. But again I won't block the PR on this.